specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -3685,6 +3859,67 @@
     }
   },
   "definitions": {
+    "ComputeOperationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ComputeOperationValue"
+          },
+          "description": "The list of compute operations"
+        }
+      },
+      "description": "The List Compute Operation operation response."
+    },
+    "ComputeOperationValue": {
+      "properties": {
```

Comment:
mark all properties as readOnly, same for next type

Reply:
(since they are only ever received from the service, not sent to it)

Reply:
fix incoming